Sources/Brave/WebFilters/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -45,12 +45,8 @@ enum BlockingStrength: String {
 class ContentBlockerHelper {
   private(set) weak var tab: Tab?
   
-  /// The rule types and source types that are currently loaded in this tab
-  private(set) var loadedRuleTypeWithSourceTypes: Set<ContentBlockerManager.RuleTypeWithSourceType> = []
-  /// The rule types with their source types that should be  loaded in this tab
-  var ruleListTypes: Set<ContentBlockerManager.RuleTypeWithSourceType> = [] {
-    didSet { reloadNeededRuleLists() }
-  }
```

Comment:
You could have just `@MainActor` isolated this property instead of make a function

Reply:
Well its actually the change was more than just @MainActor so it was a new method. I generally dislike using vars for more complex things but I can make it a var if you wish.